MediaAdminBundle/Transformer/MediaTransformer.php

```diff
@@ -70,19 +72,15 @@ public function transform($mixed)
 
         $facade->addLink('_self_select', $mixed->getId());
 
-        if ($this->authorizationChecker->isGranted(TreeFolderPanelStrategy::ROLE_ACCESS_UPDATE_MEDIA)) {
-            $facade->addLink('_self_crop', $this->generateRoute('open_orchestra_media_admin_media_crop', array(
-                'mediaId' => $mixed->getId()
-            )));
-        }
+        $facade->addLink('_self_crop', $this->generateRoute('open_orchestra_media_admin_media_crop', array(
```

Comment:
It's like the self_form for nodes, we have to show the crop form disabled if the user don't have the right to update. So we need the link.